src/main/java/com/puppycrawl/tools/checkstyle/filters/PatchXpathFilterElement.java

```diff
@@ -73,26 +78,99 @@ private boolean isFileNameMatching(TreeWalkerAuditEvent event) {
     private boolean isLineMatching(TreeWalkerAuditEvent event) {
         boolean result = false;
         if (event.getLocalizedMessage() != null) {
-            for (List<Integer> singleLineRangeList : lineRangeList) {
-                final int startLine = singleLineRangeList.get(0) + 1;
-                final int endLine = singleLineRangeList.get(1) + 1;
-                final int currentLine = event.getLine();
-                result = currentLine >= startLine && currentLine < endLine;
-                if (result) {
-                    break;
-                }
+            result = lineMatching(event.getLine());
+        }
+        return result;
+    }
+
+    private boolean lineMatching(int currentLine) {
+        boolean result = false;
+        for (List<Integer> singleLineRangeList : lineRangeList) {
+            final int startLine = singleLineRangeList.get(0) + 1;
+            final int endLine = singleLineRangeList.get(1) + 1;
+            result = currentLine >= startLine && currentLine < endLine;
+            if (result) {
+                break;
             }
         }
         return result;
     }
 
     /**
-     * Is matching by xpath query.
+     * Is matching by context strategy.
      *
      * @param event event
      * @return true if it is matching or not set.
      */
-    private boolean isXpathQueryMatching(TreeWalkerAuditEvent event) {
-        return true;
+    private boolean isContextStrategyCheck(TreeWalkerAuditEvent event) {
```

Comment:
method name is incorrect, please rename it. Method's comment is better - `isMatchingByContextStrategy()` or `matchesByContextStrategy`, etc

Reply:
done